### PR TITLE
Kv preflight

### PR DESCRIPTION
**Pull request by briankassouf**

### CODE REVIEW

File: command/kv_put.go [hunk 1]
```diff
@@ -97,14 +97,19 @@ func (c *KVPutCommand) Run(args []string) int {
 		stdin = c.testStdin
 	}
 
-	if len(args) < 1 {
+	switch {
+	case len(args) < 1:
 		c.UI.Error(fmt.Sprintf("Not enough arguments (expected 1, got %d)", len(args)))
```
Comment (jefferai) [inline command/kv_put.go]: This "expected 1" isn't true anymore, because if they provide 1 they hit the next error case.

Comment (briankassouf) [reply command/kv_put.go]: Good point!

File: vault/logical_system.go [hunk 2]
```diff
@@ -3415,25 +3472,98 @@ func (b *SystemBackend) pathInternalUIMountsRead(ctx context.Context, req *logic
...
+		},
+	}
+
+	// Load the ACL policies so we can walk the prefix for this mount
+	acl, _, entity, err := b.Core.fetchACLTokenEntryAndEntity(req)
+	if err != nil {
+		return nil, err
+	}
+	if entity != nil && entity.Disabled {
+		return nil, logical.ErrPermissionDenied
```
Comment (calvn) [inline vault/logical_system.go]: Should this return `logical.ErrEntityDisabled` instead?

Comment (jefferai) [reply vault/logical_system.go]: Yeah, should

Comment (briankassouf) [reply vault/logical_system.go]: I was trying not to use different error message so as not to leak what mounts exist. And i wanted the ACL check to happen after making sure it's a valid mount

Comment (jefferai) [reply vault/logical_system.go]: Ah, ok.

**Pull request merged**